<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
                      "http://www.w3.org/TR/html4/strict.dtd">
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<meta name="generator" content="hypermail 2.1.5, see http://www.hypermail.org/">
<title>SL4: Re: ESSAY: Program length, Omega and Friendliness</title>
<meta name="Author" content="William Pearson (wil.pearson@gmail.com)">
<meta name="Subject" content="Re: ESSAY: Program length, Omega and Friendliness">
<meta name="Date" content="2006-02-23">
<style type="text/css">
body {color: black; background: #ffffff}
h1.center {text-align: center}
div.center {text-align: center}
</style>
</head>
<body>
<h1>Re: ESSAY: Program length, Omega and Friendliness</h1>
<!-- received="Thu Feb 23 16:13:34 2006" -->
<!-- isoreceived="20060223231334" -->
<!-- sent="Thu, 23 Feb 2006 23:13:32 +0000" -->
<!-- isosent="20060223231332" -->
<!-- name="William Pearson" -->
<!-- email="wil.pearson@gmail.com" -->
<!-- subject="Re: ESSAY: Program length, Omega and Friendliness" -->
<!-- id="ab5bcc90602231513k288b5d17g@mail.gmail.com" -->
<!-- charset="ISO-8859-1" -->
<!-- inreplyto="43FDDB70.3010803@pobox.com" -->
<!-- expires="-1" -->
<p>
<strong>From:</strong> William Pearson (<a href="mailto:wil.pearson@gmail.com?Subject=Re:%20ESSAY:%20Program%20length,%20Omega%20and%20Friendliness"><em>wil.pearson@gmail.com</em></a>)<br>
<strong>Date:</strong> Thu Feb 23 2006 - 16:13:32 MST
</p>
<!-- next="start" -->
<ul>
<li><strong>Next message:</strong> <a href="14218.html">David Hart: "Re: ESSAY: Program length, Omega and Friendliness"</a>
<li><strong>Previous message:</strong> <a href="14216.html">Joel Pitt: "Re: ESSAY: Program length, Omega and Friendliness"</a>
<li><strong>In reply to:</strong> <a href="14204.html">Eliezer S. Yudkowsky: "Re: ESSAY: Program length, Omega and Friendliness"</a>
<!-- nextthread="start" -->
<li><strong>Next in thread:</strong> <a href="14202.html">William Pearson: "Re: ESSAY: Program length, Omega and Friendliness"</a>
<!-- reply="end" -->
<li><strong>Messages sorted by:</strong> 
<a href="date.html#14217">[ date ]</a>
<a href="index.html#14217">[ thread ]</a>
<a href="subject.html#14217">[ subject ]</a>
<a href="author.html#14217">[ author ]</a>
<a href="attachment.html">[ attachment ]</a>
</ul>
<hr>
<!-- body="start" -->
<p>
On 23/02/06, Eliezer S. Yudkowsky &lt;<a href="mailto:sentience@pobox.com?Subject=Re:%20ESSAY:%20Program%20length,%20Omega%20and%20Friendliness">sentience@pobox.com</a>&gt; wrote:
<br>
<em>&gt; Alejandro Dubrovsky wrote:
</em><br>
<em>&gt; &gt; On Thu, 2006-02-23 at 06:39 +0200, Mikko Särelä wrote:
</em><br>
<em>&gt; &gt;
</em><br>
<em>&gt; &gt;&gt;Certainly, though we should probably not consider a gene to be one bit,
</em><br>
<em>&gt; &gt;&gt;but more like a software component. Still we get an upper limit on the
</em><br>
<em>&gt; &gt;&gt;complexity by considerint that the human genome has 3 billion DNA base
</em><br>
<em>&gt; &gt;&gt;pairs. Since there are four different possible base pairs that means 2
</em><br>
<em>&gt; &gt;&gt;bits of information giving us a maximum of 6 Gigabits ~ 700-800 Megabytes.
</em><br>
<em>&gt; &gt;&gt;
</em><br>
<em>&gt; &gt;&gt;And that is very much overestimated by assuming that each base pair
</em><br>
<em>&gt; &gt;&gt;actually codes useful information about brain structure. Overestimated for
</em><br>
<em>&gt; &gt;&gt;the reason that no one should be able to claim that the complexity is
</em><br>
<em>&gt; &gt;&gt;_higher_ than that. It is most certainly lower than that, probably on the
</em><br>
<em>&gt; &gt;&gt;order of at least two decades (which would place it at 7-8MB) [reason
</em><br>
<em>&gt; &gt;&gt;being at least 90% seems to be junk, not all genes code for brain
</em><br>
<em>&gt; &gt;&gt;structure, and we know that a base pair does not code for 2 bits of
</em><br>
<em>&gt; &gt;&gt;information (similarly claiming that a gene codes for only 1 bit of
</em><br>
<em>&gt; &gt;&gt;information, if two different alleles exists is false idea).
</em><br>
<em>&gt; &gt;&gt;
</em><br>
<em>&gt; &gt;&gt;Anything wrong with my analysis?
</em><br>
<em>&gt; &gt;
</em><br>
<em>&gt; &gt; Someone could claim that the optional methylation of the C base is
</em><br>
<em>&gt; &gt; another state, ie each base pair having 5 options.  Then you can
</em><br>
<em>&gt; &gt; multiply by 3 to tell whether the stretch of DNA is in A-DNA, B-DNA or
</em><br>
<em>&gt; &gt; Z-DNA form (obvious overestimation there since the stretches cannot be
</em><br>
<em>&gt; &gt; one base long, and not all stretches can form Z-DNA). Then you can add 3
</em><br>
<em>&gt; &gt; to that multiplier to tell whether the DNA is hanging loose, it's
</em><br>
<em>&gt; &gt; wrapped around a histone, it's hanging between histones, or are more
</em><br>
<em>&gt; &gt; tightly packed (the tightness of the packing depends on the acethylation
</em><br>
<em>&gt; &gt; of the histones, but past some level of packing i'm assuming they are
</em><br>
<em>&gt; &gt; just not read at all, so there's probably no need to make the multiplier
</em><br>
<em>&gt; &gt; acethylation-level dependent) (again, gross overestimation since minimum
</em><br>
<em>&gt; &gt; size for each of these stretches is &gt;&gt; 1) (also, add, not multiply to
</em><br>
<em>&gt; &gt; the multiplier since these states afaik do not get a choice of whether
</em><br>
<em>&gt; &gt; they are in A-,B- or Z-). That'd be 5 x 6 = 30 choices per base.  So
</em><br>
<em>&gt; &gt; that would be lg(30) x 3 billion about 15 billion bits, slightly under 2
</em><br>
<em>&gt; &gt; gigabytes as upper limit.
</em><br>
<em>&gt;
</em><br>
<em>&gt; Heh.  Unfortunately, this potential extra data storage buys you nothing
</em><br>
<em>&gt; whatsoever in the way of information.
</em><br>
<em>&gt;
</em><br>
<em>&gt; Natural selection pressure on mammals maxes out at O(1) bits (of Shannon
</em><br>
<em>&gt; information) per generation.  See
</em><br>
<em>&gt; <a href="http://dspace.dial.pipex.com/jcollie/sle/">http://dspace.dial.pipex.com/jcollie/sle/</a>, though the argument is older
</em><br>
<em>&gt; than that - it appears for example in George Williams's &quot;Adaptation and
</em><br>
<em>&gt; Natural Selection&quot;.
</em><br>
<em>&gt;
</em><br>
<em>&gt; Bases mutate at a rate of 1e-8 per base per generation.
</em><br>
<p><em>&gt; Thus natural selection cannot maintain more than 1e8 bits of information
</em><br>
<em>&gt; against the pressure of degenerative mutation.
</em><br>
<p>This assumes that what is important over evolutionary time scales is
<br>
important for creating an individual or for an individuals function.
<br>
<p>For example the cone snail has a higher than average rate of mutation
<br>
in regions of the genome that encode for the toxins*. This implies
<br>
that over evolutionary time it doesn't matter so much what the base
<br>
pairs are. They hold little information that would be useful for
<br>
future generations.
<br>
<p>So lets say we found a shorter way of representing one of these myriad
<br>
toxins, would we expect a genetically altered cone snail with this
<br>
gene rather than the longer gene to become more dominant due to
<br>
increased efficiency of copying of the genome (less base pairs less
<br>
energy needed to copy it). Probably not because as the shorter length
<br>
genome became dominant it would become something that would be adapted
<br>
to by the snails prey and because it doesn't have the storage to store
<br>
the random variations it would not be able to create the new toxins.
<br>
<p>Because while it doesn't matter exactly which toxin it is, it matters
<br>
that it is different from the rest of the population. And so you need
<br>
the storage in the genome to store the difference.
<br>
<p>So rather than the bit being stored as a single base pair it is stored
<br>
in the choice between a long sequence of base pairs and the mechanisms
<br>
for increasing mutation in that area and a more shorter but more
<br>
static genome.
<br>
<p>&nbsp;&nbsp;Will Pearson
<br>
<p><p>*<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10415486&amp;dopt=Abstract">http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10415486&amp;dopt=Abstract</a>a<br>
<p>Or google for cone snail toxin mutation for more information
<br>
Also see the book Darwin in the Genome
<br>
<!-- body="end" -->
<hr>
<ul>
<!-- next="start" -->
<li><strong>Next message:</strong> <a href="14218.html">David Hart: "Re: ESSAY: Program length, Omega and Friendliness"</a>
<li><strong>Previous message:</strong> <a href="14216.html">Joel Pitt: "Re: ESSAY: Program length, Omega and Friendliness"</a>
<li><strong>In reply to:</strong> <a href="14204.html">Eliezer S. Yudkowsky: "Re: ESSAY: Program length, Omega and Friendliness"</a>
<!-- nextthread="start" -->
<li><strong>Next in thread:</strong> <a href="14202.html">William Pearson: "Re: ESSAY: Program length, Omega and Friendliness"</a>
<!-- reply="end" -->
<li><strong>Messages sorted by:</strong> 
<a href="date.html#14217">[ date ]</a>
<a href="index.html#14217">[ thread ]</a>
<a href="subject.html#14217">[ subject ]</a>
<a href="author.html#14217">[ author ]</a>
<a href="attachment.html">[ attachment ]</a>
</ul>
<!-- trailer="footer" -->
<hr>
<p><small><em>
This archive was generated by <a href="http://www.hypermail.org/">hypermail 2.1.5</a> 
: Wed Jul 17 2013 - 04:00:56 MDT
</em></small></p>
</body>
</html>
